src/app/components/partials/swagger-endpoint-item/swagger-endpoint-item.component.ts

```diff
@@ -11,7 +11,8 @@ export class SwaggerEndpointItemComponent implements OnInit {
   showDescription = false;
   showApiInfo = false;
 
-  constructor() { 
+  constructor() {
+    console.log(this.api);
```

Comment:
If this was added for test purposes, would it be better to remove it before merge?

Reply:
thanks I'll remove it